src/components/DiffView/index.tsx

```diff
@@ -225,7 +226,10 @@ export class DiffViewBase extends React.Component<Props> {
         )}
 
         {globalLinterMessages.length ? (
-          <div className={styles.globalLinterMessages}>
+          <div
+            className={styles.globalLinterMessages}
+            id={getCodeLineAnchorID(0)}
```

Comment:
If I understand correctly, the [`CodeView` counterpart](https://github.com/mozilla/addons-code-manager/blob/0e7150a6e4932b7afecfc67f5c859ed4f1eb0e63/src/components/CodeView/index.tsx#L76) could be rewritten to look similar to the `DiffView` one, i.e. global linter messages nested in a div. If so, then we should introduce a common component:

```jsx
<GlobalLinterMessages className={'...'}>
  {globalLinterMessages}
</GlobalLinterMessages>
```

I'm not so worried about duplicating this one line of code (the `id` assignment) but I'm more worried that one of these could go out of sync since it's not immediately clear what `getCodeLineAnchorID(0)` is doing.

If that sounds reasonable, could you file a follow-up to use a shared component?

Reply:
Moved to https://github.com/mozilla/addons-code-manager/issues/872